dts/bindings/cpu/arm,cortex-r7.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2020 IoT.bzh
```

Comment:
2021?

Reply:
Indeed, will refresh the Copyright thanks.